internal/module/logger/format.go

```diff
@@ -0,0 +1,55 @@
+package logger
+
+import (
+	"log/slog"
+	"os"
+)
+
+func SetupLogger(logLevel string, logFormat string, addSource bool) *slog.Logger {
+	level := StringToLevel(logLevel)
+	var (
+		handler  slog.Handler
+		replAttr func(groups []string, a slog.Attr) slog.Attr
+	)
+
+	// TODO hide token values
```

Comment:
@andrsj could you please remove it? 
if we need just a reminder - let's better create a ticket for that